src/index.ts

```diff
@@ -109,3 +109,49 @@ export const compare = memoize((versionA: string | null, versionB: string | null
 	}
 	return versionA.localeCompare(versionB);
 }, (a: string, b: string) => `${a} && ${b}`);
+
+/**
+ * @summary Return the major version number
+ * @name compare
```

Comment:
I guess that this should say `@name major` and everything else will change accordingly.